self-tests.yml

```diff
@@ -119,6 +119,8 @@ variables:
         params:
           directory: gopath/src/github.com/evergreen-ci/evergreen
       - func: set-up-mongodb
+      - func: run-make
+        vars: { target: "set-var" }
```

Comment:
it doesn't matter, but I think you can just combine these three run-make operations, and just pass all three targets in a space separated string to run-make.

Reply:
fixed